### DISCUSSION TITLE
Preview Button Not Working on Windows 11 with CUDA Execution Provider

### DISCUSSION THREAD

**Discussion by princecrown1019** [hacksider/Deep-Live-Cam#667] [General]
Hi,

I've encountered an issue with running the Dive-Live-Cam project on my system. While I can start the project successfully, the preview button appears to be non-functional and just empty preview screen.

i run the following command:
`python run.py --execution-provider cuda`

Clicking the preview button should show the live camera feed with face swapping applied. When I clicked the preview button, I can see the empty preview window is opened but there's an error as shown in the image below. 
![Screenshot_3](https://github.com/user-attachments/assets/f71b0bc9-d290-48c7-9841-34199048602c)

Any guidance or suggestions for troubleshooting would be greatly appreciated.

Thank you!

**Comment by KRSHH** [comment]
Preview button is not for live webcam, but for pre-recorded images and photos.
